db/migrate/20171114160332_devise_otp_add_to_admin_users.rb

```diff
@@ -0,0 +1,32 @@
+class DeviseOtpAddToAdminUsers < ActiveRecord::Migration[5.1]
+  def up
```

Comment:
seems like this could have been a `def change` ?

Reply:
Tried that, no dice. So I'll keep the up/down methods here.

See below:

```
== 20171114160332 DeviseOtpAddToAdminUsers: reverting =========================
rake aborted!
StandardError: An error has occurred, this and all later migrations canceled:



This migration uses execute, which is not automatically reversible.
To make the migration reversible you can either:
1. Define #up and #down methods in place of the #change method.
2. Use the #reversible method to define reversible behavior.


/home/joel/Projects/michigan-benefits/db/migrate/20171114160332_devise_otp_add_to_admin_users.rb:15:in `block in change'
/home/joel/Projects/michigan-benefits/db/migrate/20171114160332_devise_otp_add_to_admin_users.rb:3:in `change'
/home/joel/.rbenv/versions/2.4.2/bin/bundle:23:in `load'
/home/joel/.rbenv/versions/2.4.2/bin/bundle:23:in `<main>'
ActiveRecord::IrreversibleMigration:

This migration uses execute, which is not automatically reversible.
To make the migration reversible you can either:
1. Define #up and #down methods in place of the #change method.
2. Use the #reversible method to define reversible behavior.


/home/joel/Projects/michigan-benefits/db/migrate/20171114160332_devise_otp_add_to_admin_users.rb:15:in `block in change'
/home/joel/Projects/michigan-benefits/db/migrate/20171114160332_devise_otp_add_to_admin_users.rb:3:in `change'
/home/joel/.rbenv/versions/2.4.2/bin/bundle:23:in `load'
/home/joel/.rbenv/versions/2.4.2/bin/bundle:23:in `<main>'
Tasks: TOP => db:rollback
(See full trace by running task with --trace)
```